molgenis-data-mysql/src/test/java/org/molgenis/data/importer/EmxImportServiceTest.java

```diff
@@ -72,35 +80,35 @@ public void testValidationReport() throws IOException, InvalidFormatException, U
 		EntitiesValidationReport report = importer.validateImport(f, source);
 
 		// SheetsImportable
-		Assert.assertEquals(report.getSheetsImportable().size(), 4);
+		AssertJUnit.assertEquals(report.getSheetsImportable().size(), 4);
```

Comment:
Why the switch? (If you do need to switch, you probably also need to switch the argument order around. testng assert works with (actual, expected) while junit assert uses (expected, actual)